aws-predictions/src/main/java/com/amplifyframework/predictions/aws/http/LivenessWebSocket.kt

```diff
@@ -135,6 +139,16 @@ internal class LivenessWebSocket(
             }
         }
 
+        init {
+            // If the client session is requesting a 1.0.0 FaceMovementAndLight challenge, the backend won't return a
+            // ChallengeEvent so we have to set the challengeType manually
+            clientSessionInformation.challengeVersions.forEach {
+                if (it.compareType(Challenge.FaceMovementAndLightChallenge("1.0.0"))) {
+                    challengeType = FaceLivenessChallengeType.FaceMovementAndLightChallenge
```

Comment:
What if the client asks for 1.00 FaceMovementAndLight and then a NoLightChallenge in the array? Is that valid? If so, this is doing a loop and setting it if it sees a single item asking for it.